js/apps/admin-ui/src/groups/GroupsSection.tsx

```diff
@@ -87,7 +87,7 @@ export default function GroupsSection() {
           const group =
             i !== "search"
               ? await fetchAdminUI<GroupRepresentation | undefined>(
-                  "ui-ext/groups/" + i,
+                  "groups/" + i,
```

Comment:
Same as with the other API calls, if this is now part of the official API it should be in the Admin Client instead.